README.rst

```diff
@@ -290,26 +287,22 @@ A more complete example of a custom type might be:
 ...     return yesno_mapping[text.lower()]
 
 
-Potential Gotchas
------------------
-
-`parse()` will always match the shortest text necessary (from left to right)
-to fulfil the parse pattern, so for example:
+If the type converter ``pattern`` uses regex-grouping (with parenthesis),
+you should indicate this by using the optional ``regex_group_count`` parameter
+in the ``with_pattern()`` decorator:
 
->>> pattern = '{dir1}/{dir2}'
->>> data = 'root/parent/subdir'
->>> parse(pattern, data).named
-{'dir1': 'root', 'dir2': 'parent/subdir'}
-
-So, even though `{'dir1': 'root/parent', 'dir2': 'subdir'}` would also fit
-the pattern, the actual match represents the shortest successful match for
-`dir1`.
+>>> @with_pattern(r'((\d+))', regex_group_count=2)
+... def parse_number2(text):
+...    return int(text)
+>>> parse('Answer: {:Number2} {:Number2}', 'Answer: 42 43', dict(Number2=parse_number2))
+<Result () (42, 43)>
 
+Otherwise, this may cause parsing problems with unnamed/fixed parameters.
 ----
 
 **Version history (in brief)**:
 
-- 1.8.2 add documentation for including braces in format string
```

Comment:
1.8.2 has been released. This patch should create version 1.8.3

Reply:
It did not change this line AFAIK and thought you would add a line for version 1.8.3.
As I said the "README" file is automatically changed/updated when you run ``python setup.py sdist`` and I run just that to generate the "README" from the docstring in the ``parse`` module. I had the impression that both are out-of-sync.